GoogleSignIn/Sources/GIDSignInInternalOptions.m

```diff
@@ -36,7 +36,6 @@ + (instancetype)defaultOptionsWithConfiguration:(nullable GIDConfiguration *)con
   GIDSignInInternalOptions *options = [[GIDSignInInternalOptions alloc] init];
   if (options) {
     options->_interactive = YES;
-    options->_continuation = NO;
```

Comment:
Why is this getting removed?

Reply:
I removed this since we won't have to worry about the "continuation" property for the VwG flow and will only be used for the EMM flow. I thought since we won't make use of this in our flow then it wouldn't need to be filled. Should 'NO' still be passed to be safe?

Reply:
Sgtm, done!